src/main/java/mat/server/LoginServiceImpl.java

```diff
@@ -111,6 +107,16 @@ public LoginModel isValidUser(String userId, String password, String oneTimePass
 		LoginModel loginModel = loginCredentialService.isValidUser(userId, password, oneTimePassword,session.getId());
 		return loginModel;
 	}
+
+	@Override
+	public LoginModel getUserDetailsByHarpId(String harpId, String accessToken) {
+		logger.info("getUserDetailsByHarpId::harpId::" + harpId);
+		HttpSession session = getThreadLocalRequest().getSession();
+//		if(session.getAttribute("accessToken") == null ) { //FIXME Attribute currently unused. Might be useful to determine the token type.

```

Comment:
maybe remove this commented out code?

Reply:
Done. Never found a solid use case for it.